app/searches/services_search.rb

```diff
@@ -6,6 +6,9 @@ class ServicesSearch
 
   attribute :tags, type: String
   attribute :page, type: String
+  attribute :archived_at, type: Date
```

Comment:
You don't need to add search attributes if you're not using them in the search

Reply:
Ya i'll remove this, I added this thinking I needed it for achiving services but im handling this on the model level instead now.